### PR TITLE
Use bootstrap workspaces for clinician admin

### PR DESCRIPTION
**Pull request by paulfalgout**
Previous we were using the workspaces of the current user

Shortcut Story ID: [sc-34256]

This'll need tests

### CODE REVIEW

File: src/js/services/bootstrap.js [hunk 1]
```diff
@@ -72,7 +73,10 @@ export default App.extend({
     return this.organization;
   },
   getDirectories() {
-    return this.directories;
+    return this.directories.clone();
```
Comment (paulfalgout) [inline src/js/services/bootstrap.js:76]: not really part of the PR, but any collection from bootstrap should really be cloned..

File: src/js/views/globals/patient-modal/patient-modal_views.js [hunk 2]
```diff
@@ -212,8 +213,10 @@ const PatientModal = View.extend({
     });
   },
   showWorkspacesComponent() {
+    const currentUser = Radio.request('bootstrap', 'currentUser');
     const workspacesManager = this.showChildView('workspaces', new WorkspacesManagerComponent({
       member: this.model,
+      workspaces: currentUser.getWorkspaces(),
```
Comment (paulfalgout) [inline src/js/views/globals/patient-modal/patient-modal_views.js:219]: patient modal is currently still just the workspaces the user is in..

**Review by nmajor25 — APPROVED** [review]
